src/process/fork.rs

```diff
@@ -33,7 +35,13 @@ pub fn clone(cb: CloneCb, clone_flags: sched::CloneFlags) -> Result<Pid> {
         rlim_max: 0,
     };
     unsafe { Errno::result(libc::getrlimit(libc::RLIMIT_STACK, &mut rlimit))? };
-    let default_stack_size = rlimit.rlim_cur as usize;
+
+    // mmap will return ENOMEM if stack size is unlimited
+    let default_stack_size = if rlimit.rlim_cur != u64::MAX {
+        rlimit.rlim_cur as usize
+    } else {
+        DEFAULT_STACK_SIZE
```

Comment:
Could you have it output using `log::info!` with a reason to use the default stack size?

Reply:
done